src/Illuminate/Support/Str.php

```diff
@@ -1827,6 +1827,24 @@ public static function ucfirst($string)
         return static::upper(static::substr($string, 0, 1)).static::substr($string, 1);
     }
 
+    /**
+     * Capitalize the first character of each word in a string.
+     *
+     * @param  string  $string
+     * @param  string  $delimiter
+     * @return string
+     */
+    public static function ucwords($string, $delimiter = ' ')
```

Comment:
Is there a reason why `ucwords()` defaults to `' \t\r\n\f\v'` for `$delimiter`, while `Str::ucwords()` only defaults to `' '` in that case?

Also, to make the word-splitting safe, this should be the default:
```php
    /**
     * Capitalize the first character of each word in a string.
     *
     * @param  string  $string
     * @param  string|null  $delimiter
     * @return string
     */
    public static function ucwords($string, $delimiter = null)
    {
        $words = $delimiter === null
            ? mb_split('\s+', $value)
            : explode($delimiter, $string);

        foreach ($words as &$word) {
            $word = static::ucfirst($word);
        }

        return implode($delimiter ?? ' ', $words);
    }
```